packages/manager/modules/web-paas/src/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,25 @@
+{
+  "web_paas_title": "Vos projets Web PaaS powered by Platform.sh",
+  "web_paas_project": "Projet",
+  "web_paas_region": "Région",
+  "web_paas_status": "Statut du projet",
+  "web_paas_plan_type": "Gamme",
+  "web_paas_plan_rennewal_date": "Date de renouvellement",
+  "web_paas_plan_vcpu": "vCPU",
+  "web_paas_plan_env": "Environnement",
+  "web_paas_plan_storage": "Stockage",
+  "web_paas_open_in_web_paas_console": "Accéder à Web PaaS",
+  "web_paas_goto_dashbboard": "Voir le tableau de bord",
+  "web_paas_cancel_project": "Supprimer le projet",
+  "web_paas_create_project": "Nouveau projet",
+  "web_paas_project_status_ACTIVE": "Activé",
+  "web_paas_project_status_ERROR": "Erreur",
+  "web_paas_project_status_PENDING": "En cours",
+  "web_paas_project_status_CANCELLED": "Annulée",
+  "web_paas_project_status_CANCELLING": "Annuler en cours",
```

Comment:
Probably mistake in this phrase.

Reply:
```suggestion
  "web_paas_project_status_CANCELLING": "Annulation en cours",
```

Reply:
Hi @mohammed-zahaf ,

I will create a new ticket to handle translation issues. Since it is going to prod tomorrow, i cannot raise a translation request for this and get all translations in time for prod.